src/main/java/edu/cornell/kfs/fp/service/impl/RecurringDisbursementVoucherDocumentServiceImpl.java

```diff
@@ -409,8 +417,117 @@ public Object call() throws WorkflowException {
             throw new RuntimeException("cancelDVAsSystemUser() Unable to cancel DV: " + dvDocumentNumber, e);
         }
     }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public boolean autoApproveDisbursementVouchersSpawnedByRecurringDvs() {
+        LOG.info("autoApproveDisbursementVouchersSpawnedByRecurringDvs: Entered.");
+        int approvalCount = 0;
+        int errorCount = 0;
+        Collection <String> dvDocIds = getRecurringDisbursementVoucherSearchDao().findSavedDvIdsSpawnedByRecurringDvForCurrentAndPastFiscalPeriods(getCurrentFiscalPeriodEndDate());
+        if (dvDocIds != null) {
+            for (String dvDocId :  dvDocIds) {
+                LOG.info("Processing start for DV ID:: " + dvDocId);
```

Comment:
per kuali co standards, we should start each LOG statements with the function that we are in.